src/controller/WebController.php

```diff
@@ -360,7 +344,8 @@ public function invokeGlobalSearch($request)
     {
         $lang = $request->getLang();
         $template = $this->twig->loadTemplate('global-search.twig');
-        $this->setLanguageProperties($lang);
+        //$this->setLanguageProperties($lang);
```

Comment:
same as above, remove

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764